prow/plugins/config.go

```diff
@@ -252,25 +252,20 @@ type Approve struct {
 	// IssueRequired indicates if an associated issue is required for approval in
 	// the specified repos.
 	IssueRequired bool `json:"issue_required,omitempty"`
-
```

Comment:
Whats wrong with the empty lines?

Reply:
Because there are no new lines in other plugin configuration.